shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/schema/model/TableMetaData.java

```diff
@@ -71,7 +74,9 @@ public TableMetaData(final Collection<ColumnMetaData> columnMetaDataList, final
     private Map<String, IndexMetaData> getIndexes(final Collection<IndexMetaData> indexMetaDataList) {
         Map<String, IndexMetaData> result = new LinkedHashMap<>(indexMetaDataList.size(), 1);
         for (IndexMetaData each : indexMetaDataList) {
-            result.put(each.getName().toLowerCase(), each);
+            String lowerIndexName = each.getName().toLowerCase();
+            indexNames.add(lowerIndexName);
+            result.put(lowerIndexName, each);
```

Comment:
Please do not change `TableMetaData`, we just load, the usage is not need to change.